iot_verifier/src/poc.rs

```diff
@@ -29,7 +29,10 @@ pub const C: f64 = 2.998e8;
 pub const R: f64 = 6.371e6;
 
 /// the cadence in seconds at which hotspots are permitted to beacon
-const BEACON_INTERVAL: i64 = (10 * 60) - 10; // 10 mins ( minus 10 sec tolerance )
+const BEACON_INTERVAL: i64 = 60 * 60 * 6;
+/// a tolerance applied to beacon intervals within which beacons
+/// will be accepted
+const BEACON_INTERVAL_TOLERANCE: i64 = 10 * 60;
```

Comment:
and that's in seconds? Milis? add a comment?

Reply:
fixed here: 2900708ca135d5b2a0f7781274b0e1f8483b48c1